docs/iceberg-rest-service.md

```diff
@@ -117,6 +117,7 @@ Gravitino Iceberg REST service supports using static S3 secret key or generating
 | `gravitino.iceberg-rest.s3-role-arn`              | The ARN of the role to access the S3 data.                                                                                                                                                                          | (none)        | Yes, when `credential-provider-type` is `s3-token` | 0.7.0-incubating |
 | `gravitino.iceberg-rest.s3-external-id`           | The S3 external id to generate token, only used when `credential-provider-type` is `s3-token`.                                                                                                                      | (none)        | No                                                 | 0.7.0-incubating |
 | `gravitino.iceberg-rest.s3-token-expire-in-secs`  | The S3 session token expire time in secs, it couldn't exceed the max session time of the assumed role, only used when `credential-provider-type` is `s3-token`.                                                     | 3600          | No                                                 | 0.7.0-incubating |
+| `gravitino.iceberg-rest.s3-sts-endpoint`          | The S3 Token Service         | (none)        | No                                                 | 0.8.0-incubating |
```

Comment:
Could you add more descriptions like:
```
An alternative endpoint of the S3 token service, This could be used with s3-compatible object storage service like MINIO that has a different endpoint.
```

Reply:
Is it more user friendly to change the option name to something like
`gravitino.iceberg-rest.s3-token-service-endpoint`?

Reply:
Updated, thanks